app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(
+                f'{visitor["name"]} vaccine is expired.')
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
+            raise OutdatedVaccineError(
+                "All friends should be vaccinated")

```

Comment:
The error message for `OutdatedVaccineError` should be more descriptive. Consider changing it to: `'The vaccine for {visitor["name"]} is expired.'` to provide more context.